app/controllers/api/v1/red_line_boundary_change_requests_controller.rb

```diff
@@ -0,0 +1,24 @@
+class Api::V1::RedLineBoundaryChangeRequestsController < Api::V1::ApplicationController
+  skip_before_action :verify_authenticity_token, only: :update
+  before_action :check_token_and_set_application, only: :update
+
+  def update
+    @red_line_boundary_change_request = @planning_application.red_line_boundary_change_requests.where(id: params[:id]).first
```

Comment:
if we do a find_by_id(id: params[:id) then only one record is returned so that could be a tiny tidy up